src/app-layout/__tests__/global-breadcrumbs.test.tsx

```diff
@@ -108,15 +115,17 @@ describeEachAppLayout({ themes: ['refresh-toolbar'], sizes: ['desktop'] }, () =>
     );
   });
 
-  test('when breadcrumbs are rendered in multiple slots, the last one takes precedence', async () => {
-    await renderAsync(
+  test('when breadcrumbs are rendered in multiple slots, the one in the breadcrumbs slot takes precedence', async () => {
```

Comment:
Technically, this is a breaking change. But this the new expected one, we should be fine.